client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx

```diff
@@ -83,7 +84,7 @@ const InvestigatedPersonInfo = (props: Props) => {
         return check ? yes : no;
     };
 
-    const validateStatusReason = async (newStatusReason : string | null) => {
+    const validateStatusReason = async (newStatusReason: string | null) => {
```

Comment:
you can remove the async and await in this function

Reply:
fixed